docs/users-guide/modelsetup.rst

```diff
@@ -125,173 +126,273 @@ UV Package and Project Manager
 
-  uv run --project relative/path/to/asim_project activitysim run -c configs -o output -d data
 
+Run ActivitySim with UV
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You could also activate the virtual environment created by *uv* and run ActivitySim from any directory. 
+Activate the virtual environment created by *uv*. This option is similar to using Python's classic venv or Conda env.
 
 ::
 
+  # cd into the project directory if not already there
+  ## if you used the From PyPI option
+  cd asim_project
+  ## if you used the From Source with Lockfile option
+  cd activitysim
+
+  # Activate the virtual environment
   .venv\Scripts\activate
 
-With this command, you have activated the virtual environment created by *uv* and can run ActivitySim commands from any directory.
+Once the virtual environment is activated, you can run ActivitySim commands directly using the `activitysim` command.
+For example, run the ActivitySim commandline using the following. More information about the commandline interface is available in 
+the :ref:`Ways to Run the Model` section.
+
+::
+
+  activitysim run -c configs -o output -d data
+
+Alternatively, you can run ActivitySim commands directly using *uv* without activating the virtual environment.
+
+::
+
+  uv run activitysim run -c configs -o output -d data
+
+Common Q&A
+^^^^^^^^^^^^^^^^
+My travel demand model requires additional Python packages not included with ActivitySim. How do I add them?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can add additional packages to your *uv* project by using the `uv add` command. For example, to add the `geopandas` package, 
+run the following command within your existing *uv* project directory.
+
+::
+
+  # cd into your project directory
+  cd asim_project
+
+  # Add geopandas package
+  uv add geopandas
+
+This will add the package to your virtual environment and update the `pyproject.toml` and the `uv.lock` file to include the new package and its dependencies.
+
+If you envision having a version of Python packages that is different from the one used by ActivitySim, e.g., you need pandas 1.x for visualization (for some reason), 
+we recommend creating a separate *uv* project for your custom packages and managing them independently from ActivitySim.
+
+::
+
+  # Open Command Prompt
+  mkdir viz_project
+  cd viz_project
+  uv init
+  uv add pandas==1.5.3
+
+Many agencies use commercial software that have Python APIs and dependencies that may conflict with ActivitySim dependencies. 
+In such cases, we also recommend creating a separate *uv* project for the commercial software and managing them independently from ActivitySim.
+
+::
+
+  # Open Command Prompt
+  mkdir emme_project
+  cd emme_project
+  uv init --python 2.7
+  # Then copy the emme.pth file (provides EMME API handshakes) from the Emme installation directory to emme_project/.venv/Lib/site-packages/
```

Comment:
Suggest we explicitly state here that users can easily switch between different UV environments by just activating the one they want to use. (Do you need to deactivate your current environment first?)